src/main/java/executor/service/service/impl/ScenarioSourceListenerImpl.java

```diff
@@ -37,4 +33,22 @@ private void validateScenarios(List<Scenario> scenarios) {
             throw new IllegalArgumentException("List cannot be null or empty");
```

Comment:
**throw new IllegalArgumentException("List cannot be null or empty");** - In my opinion, this should be removed, we don’t need the program to crash. It is enough to register this event.

Reply:
how the program can even start if the scenario list would be empty? what should we execute?

Reply:
Most likely the program will not be launched without scripts.